ts/modules/data.src.ts

```diff
@@ -248,25 +248,25 @@ declare global {
 
 import U from '../parts/Utilities.js';
 const {
+    addEvent,
     defined,
     extend,
+    fireEvent,
     isNumber,
+    merge,
     objectEach,
     pick,
     splat
 } = U;
 
-import '../parts/Chart.js';
+import H from '../parts/Globals.js';
 import '../mixins/ajax.js';
+import '../parts/Chart.js';
 
 // Utilities
-var addEvent = Highcharts.addEvent,
-    Chart = Highcharts.Chart,
+var Chart = Highcharts.Chart,
     win = Highcharts.win,
```

Comment:
These two have to use the imported `H.` instead of `Highcharts.`. Maybe look for similar cases in the file, where `Highcharts.` is not used as a type but as an object.